README.adoc

```diff
@@ -14,7 +14,7 @@ image:https://api.netlify.com/api/v1/badges/5b89dd6f-1847-419c-b3be-a1650ce8992f
 </p></a>
 ++++
 
-This repository hosts the documentation content for Redpanda Platform and Redpanda Cloud.
+This repository hosts the documentation content for Redpanda Enterprise.
```

Comment:
Shouldn't we call it Redpanda Platform to align with [marketing](https://redpanda.com/what-is-redpanda)?

![2024-06-11_17-24-58](https://github.com/redpanda-data/docs/assets/45230295/b15e09e9-586e-4a1b-acab-b2fad9f6fb54)